components/scream/scripts/test_all_scream.py

```diff
@@ -108,7 +108,8 @@ def __init__(self):
             self,
             "debug_nopack_fpe",
             "debug pksize=1 floating point exceptions on",
-            DBG.CMAKE_ARGS + [("SCREAM_PACK_SIZE", "1")],
+            DBG.CMAKE_ARGS + [("SCREAM_PACK_SIZE", "1"), ("EKAT_ENABLE_FPE","True")],
+            uses_baselines=False
```

Comment:
@jgfouca This one is a bit unrelated, but on my machine it was throwing since I did not have fpe baselines. And I started wondering if we do care about baselines for FPE builds. I think in FPE builds, we are really testing code correctness (much like a memcheck build), so I removed baselines for this.

But I can roll this back if you disagree.